lib/klaviyo/apis/track.rb

```diff
@@ -0,0 +1,42 @@
+class Track < Klaviyo::Client
+  # Used for tracking events and customer behaviors
+  #
+  # @param event [String] the event to track
+  # @kwarg :id [String] the customer or profile id
+  # @kwarg :email [String] the customer or profile email
+  # @kwarg :properties [Hash] properties of the event
+  # @kwargs :customer_properties [Hash] properties of the customer or profile
+  # @kwargs :time [Integer] timestamp of the event
+  def self.track(event, kwargs = {})
+    defaults = {:id => nil,
```

Comment:
should all be a new line
```suggestion
    defaults = {
        :id => nil,
        ....
     }
```